### PR TITLE
docs(zh): update state.md about `$reset`

### PR DESCRIPTION
**Pull request by daxiongz**
In English Doc:

> In [Option Stores](/core-concepts/index.md#option-stores), you can _reset_ the state to its initial value by calling the `$reset()` method on the store:

But in Chinese Doc:

> 你可以通过调用 store 的 $reset() 方法将 state 重置为初始值。

### CODE REVIEW

**Comment by KimYangOfCat** [comment]
This is because the Chinese documents are not synchronized in time

File: packages/docs/zh/core-concepts/state.md [hunk 1]
```diff
@@ -86,7 +86,7 @@ store.count++
 
 ## 重置 state %{#resetting-the-state}%
 
-你可以通过调用 store 的 `$reset()` 方法将 state 重置为初始值。
+使用选项式 API 的用法时，你可以通过调用 store 的 `$reset()` 方法将 state 重置为初始值。
```
Comment (KimYangOfCat) [inline packages/docs/zh/core-concepts/state.md]:
```suggestion
使用[选项式 API](/zh/core-concepts/index.md#option-stores) 时，你可以通过调用 store 的 `$reset()` 方法将 state 重置为初始值。
```

**Comment by KimYangOfCat** [comment]
LGTM

**Review by awxiaoxian2020 — COMMENTED** [review]
LGTM！

**Pull request merged**